client/src/store/modules/auth/getters.js

```diff
@@ -0,0 +1,8 @@
+import * as getters from '@/store/modules/auth/types/getters';
+
+export default {
+    [getters.HAS_TOKEN]: state => !!state.token,
+    [getters.GET_TOKEN]: state => state.token,
+    [getters.IS_LOGGED_IN]: state => !!state.isLoggedIn,
+    [getters.GET_LOGGED_USER]: state => state.user
```

Comment:
if user is not logged in, you can save null, and then check if user exists or not, it is how you'll detect the authorization